back/app/models/idea.rb

```diff
@@ -111,6 +113,7 @@ class Idea < ApplicationRecord
   validates :proposed_budget, numericality: { greater_than_or_equal_to: 0, if: :proposed_budget }
 
   validate :validate_creation_phase
+  validate :not_published_in_non_public_status
```

Comment:
I agree the double negation in the name is confusing. But when there is a submitted or draft input without a public status, the validation will allow it. Perhaps I could make the validation more strict to also cover those cases? I was a bit afraid to do this because it felt more risky to break existing data like survey responses.

Reply:
The main case I wanted to avoid here is having an input in prescreening with the published input status. Because of the data duplication, I felt like the validation of this specific case would be the most desirable.

Reply:
Yes, in that case the double negation probably makes sense